app/views/items/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= form_for @item, :url => dashboard_items_new_path(@item) do |f| %>
```

Comment:
I think the standard path for this would be dashboard_items_path, with a method of post (which should be default here)

Reply:
Hmm - the user story called for the /new:
```When I visit my items page
And I click on the link to add a new item
My URI route should be "/dashboard/items/new"```
Should I change it? Or keep it as is?

Reply:
Changed!

Reply:
Yep - the order in the routes file was messing with the path helper prefix.  Moved it and it is all good now!